src/Diagnostics.DataProviders/DataSourcesConfiguration.cs

```diff
@@ -19,5 +19,7 @@ public class DataSourcesConfiguration
         public AntaresMdmDataProviderConfiguration AntaresMdmConfiguration { get; set; }
 
         public NetworkingMdmDataProviderConfiguration NetworkingMdmConfiguration { get; set; }
+
+        public CompilerHostAuthConfiguration CompilerHostAuthConfiguration { get; set; }
```

Comment:
Compiler host is not a data provider. Don't think its a good idea to take dependencies on data provider project for this one. Structurally also, it will create confusion going forward.